src/lib.rs

```diff
@@ -37,9 +37,11 @@ pub struct Challenge {
 
 #[derive(Clone, Serialize, Deserialize, Debug)]
 pub struct TeePubKey {
-    pub kty: String,
     pub alg: String,
-    pub k: String,
+    #[serde(rename = "k-mod")]
+    pub k_mod: String,
+    #[serde(rename = "k-exp")]
+    pub k_exp: String,
```

Comment:
Hey @tylerfanelli!

Since we (https://github.com/confidential-containers) depend on those definitions (and incidentally [define them](https://github.com/confidential-containers/kbs/blob/main/docs/kbs_attestation_protocol.md#public-key) ;-) ), could you please add us (myself, @Xynnn007 and @jialez0) to PRs that potentially change the protocol and break our implementations?

Reply:
@sameo Oops... I'm really sorry about that. We can revert this one if needed. Also, it'd probably be a good idea adding one of you folks as CODEOWNERS here. What do you think?

Reply:
I'm in favor of @sameo to take this! :)

Reply:
> Should we send a PR for that, @slp?

@sameo Yes, please.

Reply:
> > Should we send a PR for that, @slp?
> 
> @sameo Yes, please.

Thanks, here goes the PR: #18